packages/utils/src/useI18n.ts

```diff
@@ -0,0 +1,40 @@
+import { useEffect, useState } from 'react';
+import { dynamicActivateI18n } from './dynamicActivatei18n';
+
+function detectLocale() {
+  const supportedLocales = ['en', 'nb'];
+  const defaultLocale = 'en';
+  try {
+    const htmlLocale = document.documentElement.lang;
```

Comment:
This might not work server side. Need to look into how to get the correct language here.